front/app/modules/commercial/customizable_navbar/admin/containers/NavigationSettings/HiddenNavbarItemList/Header.tsx

```diff
@@ -20,10 +19,6 @@ const Header = ({ itemsNotInNavbarPresent }: Props) => (
         <FormattedMessage {...messages.hiddenFromNavigation} />
       </SubSectionTitle>
     )}
-
-    <Button buttonStyle="cl-blue" linkTo="/admin/pages-menu/pages/new">
-      <FormattedMessage {...messages.addPageButton} />
-    </Button>
```

Comment:
Could you reason me through the deletion of this? Just to make sure we know what we're doing.

When customizable navbar module is off, we render this component, in which we won't be able to create new pages?

Reply:
And navbar will be available to all, so this file won't be needed in the future?

Reply:
It removes the button here, which would normally be next to "Other available pages" since now the create button is in the top right. 

![Screen Shot 2022-10-11 at 10 54 45](https://user-images.githubusercontent.com/3614128/195045285-a6c40a85-4d2c-404a-91bd-5a42a23d8d38.png)

Not sure I understand the feature flag difference here, which files should I look in?

Reply:
> When customizable navbar module is off, we render this component, in which we won't be able to create new pages?

This change is inside of the customizable navbar module. So, if the module is off, we cannot render it.

Reply:
I think that case would look like this, where the button is at the top, but it's hard to see every place this component loads since it's in a module

![Screen Shot 2022-10-11 at 11 06 43](https://user-images.githubusercontent.com/3614128/195047950-f53b42dc-7522-4327-b10f-97392b5dd2b1.png)

Reply:
> It removes the button here, which would normally be next to "Other available pages" since now the create button is in the top right.
> 
> ![Screen Shot 2022-10-11 at 10 54 45](https://user-images.githubusercontent.com/3614128/195045285-a6c40a85-4d2c-404a-91bd-5a42a23d8d38.png)
> 
> Not sure I understand the feature flag difference here, which files should I look in?

We have the new button in front/app/modules/commercial/customizable_navbar/admin/containers/index.tsx.

But what if the module is turned off? How can I add a new page then? I don't see the Create custom page button in front/app/containers/Admin/pagesAndMenu/index.tsx.

Ideally, we make it the same for everyone, because it'll be available to all from what I understood.

Reply:
Maybe I missed something, I'm not super deep into it anymore after a week off. Just double checking.

Reply:
I also forgot: https://citizenlabco.slack.com/archives/C2W7GP5MY/p1664474437192609

So it's available for everyone now

Reply:
And what about OS users who don't have EE?

Reply:
Ok, I'm officially confused, can you tell me where the button should go?

Reply:
I _think_ the button in front/app/modules/commercial/customizable_navbar/admin/containers/index.tsx should be in the front/app/containers/Admin/pagesAndMenu/index.tsx since everyone can create custom pages and custom page is in the core, not in a module?